interface/backend/submission/evaluator/vmck.py

```diff
@@ -75,4 +75,9 @@ def update(submission):
             log.debug(f"Submission #{submission.pk} update error: {e}")
             return "Error"
         else:
-            return response.json()["state"]
\ No newline at end of file
+            try:
```

Comment:
I think you can remove the ```try catch``` from line 68

Reply:
Done.
Unfortunately the error on the unit test remains the same.
Isn't the problem on the test method ?